drivers/wifi/nrf700x/osal/fw_if/umac_if/src/event.c

```diff
@@ -527,13 +528,73 @@ wifi_nrf_fmac_data_event_process(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 
 	switch (event) {
 	case NRF_WIFI_CMD_RX_BUFF:
+#ifdef CONFIG_NRF700X_RX_DONE_WQ_ENABLED
+		struct nrf_wifi_rx_buff *config = wifi_nrf_osal_mem_zalloc(
+			fmac_dev_ctx->fpriv->opriv,
+			sizeof(struct nrf_wifi_rx_buff));
+		if (!config) {
+			wifi_nrf_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+					      "%s: Failed to allocate memory (RX)\n",
+					      __func__);
+			status = WIFI_NRF_STATUS_FAIL;
+			break;
+		}
+		wifi_nrf_osal_mem_cpy(fmac_dev_ctx->fpriv->opriv,
+					config,
```

Comment:
Indentation

Reply:
Tried again, aligning to open brace vs. not exceeding 100 lines vs. using tabs as much as possible, its impossible to do this alignment anyother (AFAIK), so, closing this comment as checkpatch is passing.